modules/integration-tests/pom.xml

```diff
@@ -77,6 +77,19 @@
       <groupId>org.apache.hadoop</groupId>
       <artifactId>hadoop-client-api</artifactId>
     </dependency>
+    <!--LogIT uses log4j V1 APIs to gather log statemtents to veify their correctness. So here we are loading the log4j V2 api shim for V1.  -->
+    <dependency>
+      <groupId>org.apache.logging.log4j</groupId>
+      <artifactId>log4j-1.2-api</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.logging.log4j</groupId>
+      <artifactId>log4j-core</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.logging.log4j</groupId>
+      <artifactId>log4j-slf4j2-impl</artifactId>
+    </dependency>
```

Comment:
ZooKeeper used logback, which seemed to make it easier to verify logging in test code. Could use logback for tests, but use log4j-slf4j2-impl/log4j-core for distribution. But that would require changing the test. (Would be a follow-on task.)

Reply:
Yeah it would be good to figure out if there is a better way to get log message in a unit test using log4j2.  I searched around a bit and didn't find anything.  Seems like this would be a common need.